Parkinson/Resources/JSONs/StudyOverview.json

```diff
@@ -3,6 +3,7 @@
     "from_date":"December 2014",
     "to_date":"January 2015",
     "logo_name": "logo_research_institute",
+    "blog_url": "http://sagebionetworkstest.blogspot.com/feeds/posts/default?alt=rss",

```

Comment:
Yes, the test URL was intentional as that's currently the correct URL. I'll replace "sagebionetworkstest" with "study news".